lib/rules/custom-property-no-outside-root/README.md

```diff
@@ -1,6 +1,6 @@
 # custom-property-no-outside-root
 
-***Deprecated: this rule is outside the scope of stylelint's functionality. See [the release planning docs](https://stylelint.io/user-guide/release-planning/) for details.***
+***Deprecated: See [CHANGELOG](../../../CHANGELOG.md).***
```

Comment:
Is it worth pre-emptively linking to the `v7.8.0` part of the changelog here?

` (../../../CHANGELOG.md#780)`